src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
+const basePrice = 40;
+const threeDays = 3;
+const threeDaysDiscount = 20;
+const sevenDays = 7;
+const sevenDaysDiscount = 50;
+
 function calculateRentalCost(days) {
```

Comment:
The parameter name `days` is clear, but it could be more descriptive. Consider renaming it to `rentalDays` to explicitly state what the number represents.